__tests__/html/activityStatusTelemetry.ignoreStatusChange.incomingActivity.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <link href="/assets/index.css" rel="stylesheet" type="text/css" />
+    <script crossorigin="anonymous" src="/test-harness.js"></script>
+    <script crossorigin="anonymous" src="/test-page-object.js"></script>
+    <script crossorigin="anonymous" src="/__dist__/webchat-es5.js"></script>
+  </head>
+  <body>
+    <div id="webchat"></div>
+    <script>
+      run(
+        async function () {
+          const store = testHelpers.createStore();
+
+          const directLine = testHelpers.createDirectLineEmulator(store);
+          const telemetryEventTracking = [];
+          WebChat.renderWebChat(
+            {
+              directLine,
+              store,
+              styleOptions: {
+                groupTimestamp: 60000,
+                sendTimeout: 5000
+              },
+              onTelemetry: (event) => {
+                if (event?.name === 'send-status:change') {
+                  telemetryEventTracking.push(event);
+                }
+              },
+            },
+            document.getElementById('webchat')
+          );
+
+          await pageConditions.uiConnected();
+
+          await directLine.emulateIncomingActivity('Aloha!');
```

Comment:
Let's write some comments in BDD style. I was using SETUP/WHEN/THEN. It should be GIVEN/WHEN/THEN. I will slowly fix mine.

```suggestion
          // GIVEN: An incoming activity.
          await directLine.emulateIncomingActivity('Aloha!');
```

Reply:
updated